src/devices/localization2DServer/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+yarp_prepare_plugin(localization2DServer
+                    CATEGORY device
+                    TYPE localization2DServer
+                    INCLUDE localization2DServer.h
+                    DEFAULT ON)
+
+if(NOT SKIP_localization2DServer)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+yarp_add_plugin(yarp_localization2DServer localization2DServer.h
+                                   localization2DServer.cpp
+                                  ${ROS_MSG})
```

Comment:
I can't find this in the current YARP codebase, is this a leftover?

Reply:
Duplicate of the comment below. guys, not all the files are committed yet. Please be patient, ROS is used, at least on my local version :-)

Reply:
ah sorry, I mean: ROS is used, $ROS_MSG not!